tests/integration/cloudformation/engine/test_conditions.py

```diff
@@ -376,3 +376,22 @@ def test_sub_in_conditions(self, deploy_cfn_template, aws_client):
         topic_arn_with_suffix = stack.outputs["TopicWithSuffixRef"]
         aws_client.sns.get_topic_attributes(TopicArn=topic_arn_with_suffix)
         assert topic_arn_with_suffix.split(":")[-1] == f"{topic_prefix}-{region}-{suffix}"
+
+    @pytest.mark.aws_validated
+    @pytest.mark.parametrize("env", ["dev", "production"])
```

Comment:
minor: worth parametrizing the region as well? Just to check both sides of the conditional